### PR TITLE
Validate variables in SET TO ON/OFF statement

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java [hunk 1]
```diff
@@ -51,25 +54,20 @@ public ElementItem(
...
   @Override
   public String getFormattedDisplayLine() {
     StringBuilder stringBuilder = new StringBuilder(getFormattedSuffix());
-    if (picClause != null)
-      stringBuilder.append(" PIC ").append(picClause);
-    if (usageFormat != UsageFormat.UNDEFINED)
-      stringBuilder.append(" USAGE ").append(usageFormat);
-    if (StringUtils.isNoneBlank(value))
-      stringBuilder.append(" VALUE ").append(value);
+    if (picClause != null) stringBuilder.append(" PIC ").append(picClause);
```
Comment (asatklichov) [inline server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java:68]: Better to extract `PIC, VALUE`,  etc. ... as CONST, e.g.  repeatedly used in `ConditionDataName`, `IndependentDataItem` , `MultiTableDataName`, `TableDataName`..   then only remains add padding either with Java or using Guava methods

**Review by grianbrcom — APPROVED** [review]
:eagle:

File: server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java:68]: I will do it as a separate PR, maybe altogether with setting DISPLAY as a default usage.

**Review by asatklichov — APPROVED** [review]
🦅

**Review by zeibura — APPROVED** [review]
LGTM

**Review by grianbrcom — APPROVED** [review]

**Pull request merged**